index_deskop.html

```diff
@@ -0,0 +1,314 @@
+          <div class="flexbox-project-item flexbox-project-item-1">
+            <img src="images/desktop_p4.png" alt="" />
+          </div>
+          <div class="flexbox-project-item flexbox-project-item-2">
+            <h3 class="project_name">Multi-Post Stories</h3>
+            <span class="conopy">FACEBOOK</span>
+            <ul class="hug_content">
+              <li>Full Stack Dev</li>
+              <li>2015</li>
+            </ul>
+            <p class="hug_content_desc">
+              Experimental content creation feature that allows users to add to
+              an existing story over the course of a day without spamming their
+              friends.
+            </p>
+
+            <ul class="on_row">
+              <li class="languages"><a href="#">HTML </a></li>
+              <li class="languages"><a href="#">Ruby on Rails </a></li>
+              <li class="languages"><a href="#">CSS </a></li>
+              <li class="languages"><a href="#">JavaScript </a></li>
+            </ul>
+
+            <a href="" class="see_projects"> See Project </a>
+          </div>
+        </div>
+      </section>
+    </article>
+
+    <article>
+      <div class="grid-container-about p-1 m-t-3">
+        <div class="grid-item grid-item-1">
+          <header>
+            <h2 class="about_header">
+              About <br />
+              MySelf
+            </h2>
+          </header>
+
+          <div>
+            <p class="self_desc">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+
+          <div>
+            <h4 class="height_inc">LET'S CONNECT</h4>
+            <ul class="on_row">
+              <li>
+                <a href="">
+                  <img class="size" src="images/vector_1.png" alt="" />
+                </a>
+              </li>
+              <li>
+                <a href="">
+                  <img class="size" src="images/vector_5.png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="">
+                  <img class="size" src="images/vector_2.png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="">
+                  <img class="size" src="images/vector_3.png" alt=""
+                /></a>
+              </li>
+              <li>
+                <a href="">
+                  <img class="size" src="images/medium.png" alt=""
+                /></a>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <a class="see_projects" href="">Get my Resume</a>
+          </div>
+        </div>
+
+        <div class="grid-item grid-item-2">
+          <div class="flex-item flex-item-1 p-t-1 category-header">
```

Comment:
- [ ] There seems to be a gap between your About and Contact sections. Kindly remove this gap as the Figma template does not have one
> Your 
![image](https://user-images.githubusercontent.com/90556221/165942320-4ad6f760-58aa-498d-86cb-19416116aaf2.png)

> Figma
![image](https://user-images.githubusercontent.com/90556221/165942443-01ebc370-6855-4e73-b7d7-0d3da2e3710c.png)

Reply:
Corrected